disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +208,13 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """This function takes as input an integer that represents a cycle count,
+        calculates jitter and executes sleep for the calculated time.
+        The invoking function should increment 'cycle' before calling this function.
+        The minimum value 'cycle' can take is 1
+        """
```

Comment:
When I write long comment in one line I get error from the pycharm telling that the line is too long.